lib/job-simulation.rb

```diff
@@ -1,31 +1,53 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative './Stack.rb'
+require_relative './Queue.rb'
 
 class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    count = 1
+    @workers = Stack.new
+    jobs_available.times do
+      @workers.push("Worker ##{count}")
+      count += 1
+    end
+    @waiting = Queue.new
+    job_seekers.times do
+      waiting.enqueue("Worker ##{count}")
+      count += 1
+    end
```

Comment:
This works, but with ruby you can do the following (which reads cleaner):
```
jobs_available.times do |count|
  @workers.push("Worker ##{count+1}")
end
```

Figuring out how to use this same syntax for the waiting queue isn't trivial, but is similar.